akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -87,14 +87,26 @@ object ActorPath {
   final def validatePathElement(element: String, fullPath: String): Unit = {
     def fullPathMsg = if (fullPath ne null) s""" (in path [$fullPath])""" else ""
 
-    (findInvalidPathElementCharPosition(element): @switch) match {
-      case ValidPathCode =>
-      // valid
-      case EmptyPathCode =>
-        throw InvalidActorNameException(s"Actor path element must not be empty $fullPathMsg")
-      case invalidAt =>
-        throw InvalidActorNameException(s"""Invalid actor path element [$element]$fullPathMsg, illegal character [${element(
-          invalidAt)}] at position: $invalidAt. """ +
+    // TODO DOTTY:
+    // ould not emit switch for @switch annotated match since there are not enough cases
+    // (findInvalidPathElementCharPosition(element): @switch) match {
```

Comment:
Here is a problem with the Scala 3 compiler, some (not sure if all) of the `switch` annotations need to be rewritten since the compiler fails with:
`Could not emit switch for @switch annotated match since there are not enough cases`

Any help is appreciated @smarter @nicolasstucki 🙏

Reply:
Just comment out the `@switch`, it's a no-op in this case, we plan to remove that warning but that hasn't been done yet: https://github.com/lampepfl/dotty/issues/9776

Reply:
done in #29930